pytest-saas/exasol/pytest_saas/project_short_tag.py

```diff
@@ -0,0 +1,30 @@
+"""
+A "Project Short Tag" is a short abbreviation for a project.
+
+The pytest plugin for exasol-saas-api will include this short tag into the
+names of created database instances to enable identifying the origin of
+potentially long-running database instances in order to avoid unwanted costs.
+"""
+
+from pathlib import Path
+
+FILE = "error_code_config.yml"
+
+
+def read_from_yaml(dir: Path) -> str:
+    """
+    Read project-short-tag from yaml file ``FILE`` in the specified
+    directory ``dir``.
+    """
+    config_file = dir / FILE
+    if not config_file.exists():
+        return None
+    content = config_file.read_text()
+    header = False
+    for line in content.splitlines():
```

Comment:
I had guessed the error-code file is json or yaml, in that case would have parsed it that way. The would be much more reliable.

Reply:
you can use pyyaml to read yaml files

Reply:
see next push